src/index.js

```diff
@@ -90,12 +90,17 @@ class JanusAdapter {
     this.reconnectionAttempts = 0;
 
     this.publisher = null;
+    this.occupantIds = [];
     this.occupants = {};
-    this.leftOccupants = new Set();
+    window.occupants = this.occupants;
```

Comment:
^^ window

Reply:
This was actually already in here, i assume for debugging purposes.